src/components/ChartContainer.js

```diff
@@ -110,79 +112,6 @@ const ChartContainer = forwardRef(
       }
     };
 
-    const panEndHandler = () => {
```

Comment:
I love seeing all this code deleted!

Reply:
The panning actually works really well with this library